pokemon/src/main/java/gamarket/PokemonCollection.java

```diff
@@ -117,23 +120,81 @@ public int getNumPokes()
 	
 	/**
 	 * A public function printCollection of type void.

```

Comment:
See this kind of comment in a few of the javadoc comments. Not really wrong but it doesn't explain anything the code doesn't. So we don't really need it.